base/heartbeat.go

```diff
@@ -380,6 +413,8 @@ func (dh *documentBackedNodeListHandler) updateNodeList(nodeID string, remove bo
 			dh.nodeIDs = append(dh.nodeIDs, nodeID)
 		}
 
+		log.Printf("Writing nodeList document (%s): %v", dh.nodeListKey, dh.nodeIDs)
```

Comment:
Candidate for a trace log when we want to debug a test

```suggestion
		Tracef("Writing nodeList document (%s): %v", dh.nodeListKey, dh.nodeIDs)
```

Reply:
Thanks for catching - will swap to trace.